src/components/login.tsx

```diff
@@ -0,0 +1,39 @@
+import React, { CSSProperties, Fragment } from 'react';
+
+export class EmailInput extends React.Component {
+  render() {
+    return (
+      <Fragment >
+        <label htmlFor="email">Email</label>
+        <input type='text' name='email' id="email" style={inputStyles}/>
+      </Fragment>
```

Comment:
Só uma dica: o jsx/tsx tem um [syntactic sugar](https://en.wikipedia.org/wiki/Syntactic_sugar#:~:text=In%20computer%20science%2C%20syntactic%20sugar,style%20that%20some%20may%20prefer.) para esses casos
```suggestion
      <>
        <label htmlFor="email">Email</label>
        <input type='text' name='email' id="email" style={inputStyles}/>
      </>
```